### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by stewartbryson**

### CODE REVIEW

File: src/main/groovy/com/redpillanalytics/gradle/ConfluentPluginExtension.groovy [hunk 1]
```diff
@@ -70,6 +70,16 @@ class ConfluentPluginExtension {
     */
    String pipelineEndpoint = 'http://localhost:8088'
 
+   /**
+    * Username for Basic Authentication with the RESTful endpoint. Default: ''.
+    */
+   String pipelineUsername = ''
```
Comment (CHR-LeeOlsen) [inline src/main/groovy/com/redpillanalytics/gradle/ConfluentPluginExtension.groovy]: Does this need to be changed to `String pipelineUsername` like it was in the KsqlRest.groovy file?

Comment (CHR-LeeOlsen) [reply src/main/groovy/com/redpillanalytics/gradle/ConfluentPluginExtension.groovy]: I'm @Sikta. I had to create a work account for some stuff we are working on.

Comment (stewartbryson) [reply src/main/groovy/com/redpillanalytics/gradle/ConfluentPluginExtension.groovy]: > Does this need to be changed to `String pipelineUsername` like it was in the KsqlRest.groovy file?

Good catch.

Comment (stewartbryson) [reply src/main/groovy/com/redpillanalytics/gradle/ConfluentPluginExtension.groovy]: New release will be published in the next 20 minutes or so.

Comment (CHR-LeeOlsen) [reply src/main/groovy/com/redpillanalytics/gradle/ConfluentPluginExtension.groovy]: I can't thank you enough! This plugin saved us a ton of work figuring out how to deploy KSQL in our org.

**Pull request merged**